src/Microsoft.AspNet.Mvc.Localization/HtmlLocalizerFactory.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.Localization;
+using Microsoft.Framework.WebEncoders;
+
+namespace Microsoft.AspNet.Mvc.Localization
+{
+    /// <summary>
+    /// An <see cref="IHtmlLocalizerFactory"/> that creates instances of <see cref="HtmlLocalizer"/>.
+    /// </summary>
+    public class HtmlLocalizerFactory : IHtmlLocalizerFactory
```

Comment:
Why does this second way of creating `IHtmlLocalizer`s for a specific `Type` exist? That is, could we do without the factory and just activate `IHtmlLocalizer<T>` instances?